serverless/functions/serverless-functions-on-cluster-builds.adoc

```diff
@@ -0,0 +1,90 @@
+:_content-type: PROCEDURE
+[id="serverless-functions-on-cluster-builds"]
+= Building and deploying functions on-cluster
+:context: serverless-functions-on-cluster-builds
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+You can use the Knative (`kn`) CLI to initiate a function project build directly on the cluster. To build a function project in this way, the source code for your function project must exist in a Git repository branch that is accessible to your cluster.
+
+:FeatureName: {FunctionsProductName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="prerequisites_serverless-functions-on-cluster-builds"]
+== Prerequisites
+
+* xref:../../cicd/pipelines/installing-pipelines.adoc[{pipelines-title}] must be installed on your cluster.
```

Comment:
@bergerhoffer @johnwilkins I believe this was why it was done as an assembly rather than a procedure. It's a much messier UX to say you need pipelines as a prereq but then have a user have to "see additional resources" to find instructions on that, rather than just being able to include the xref in the prereqs. I would prefer to use the prereqs like [this](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies) but then it wouldn't be using the **.Prerequisites** in the module, unless I repeat them there.